connect/api/src/main/java/org/apache/kafka/connect/header/ConnectHeaders.java

```diff
@@ -44,6 +44,8 @@
  */
 public class ConnectHeaders implements Headers {
 
+    public static final String HEADER_PREFIX = "__connect";
```

Comment:
Why would we introduce this constant here? This makes it seem like all headers will have this prefix.

Reply:
That was the idea actually. To have a common prefix for all Connect system headers. Do you think it is unnecessary?

Reply:
Well, here it's part of the public API, and that I think is a bad idea. We can move that to something in the Runtime, but there's no good place ATM. I might suggest moving it to `DeadLetterQueueReporter` where it'd be used and we can refactor later if needed.